rever/activities/push_tag.xsh

```diff
@@ -0,0 +1,59 @@
+"""Activity for pushing remote tags."""
+import re
+
+from rever import vcsutils
+from rever.activity import Activity
+from rever.tools import eval_version
+
+
+class PushTag(Activity):
+    """Pushes the current tag up to a remote repository.
+
+    This activity takes the following parameters:
+
+    :$TAG_REMOTE: str or None, remote URL to push tags to.
+    :$TAG_TARGET: str or None, remote branch to push to once the tag has been made.
+        The default is None, which uses the current branch.
+    :$GITHUB_ORG str or None, GitHub org to push to if no $TAG_REMOTE
+    :$GITHUB_REPO = str or None GitHub repo to push to if no $TAG_REMOTE
```

Comment:
These two should be in another section, since they aren't bound to the kwargs.  See the CondaForge activity docs for an example.